Classes/book.rb

```diff
@@ -0,0 +1,15 @@
+require './item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state
+
+  def initialize(publisher, cover_state)
+    super(publish_date)

```

Comment:
I think `publish_date` should be passed through the constructor as an argument.

Reply:
I have made this changes 😄